flytekit/bin/entrypoint.py

```diff
@@ -316,5 +342,19 @@ def fast_execute_task_cmd(additional_distribution, dest_dir, task_execute_cmd):
     _os.system(" ".join(task_execute_cmd))
 
 
+@_pass_through.command("pyflyte-map-execute")
+@_click.option("--task-module", required=True)
```

Comment:
I would make sure this works with the new style that @wild-endeavor is thinking about, that way there will be one lesser modification. I mean if we do not need task-module/task-name then this is the time to get rid of it

Reply:
synced with Yee offline, we'll coordinate to update this